routers/api/v1/repo/action.go

```diff
@@ -45,20 +45,19 @@ func CreateOrUpdateSecret(ctx *context.APIContext) {
 	//     "$ref": "#/definitions/CreateOrUpdateSecretOption"
 	// responses:
 	//   "201":
-	//     description: response when creating a secret
+	//     description: secret created
 	//   "204":
-	//     description: response when updating a secret
+	//     description: secret updated
 	//   "400":
 	//     "$ref": "#/responses/error"
 	//   "404":
 	//     "$ref": "#/responses/notFound"
 
-	owner := ctx.Repo.Owner
 	repo := ctx.Repo.Repository
 
 	opt := web.GetForm(ctx).(*api.CreateOrUpdateSecretOption)
 
-	_, created, err := secret_service.CreateOrUpdateSecret(ctx, owner.ID, repo.ID, ctx.Params("secretname"), opt.Data)
+	_, created, err := secret_service.CreateOrUpdateSecret(ctx, 0, repo.ID, ctx.Params("secretname"), opt.Data)
```

Comment:
This maybe a break change? It wil lead some inconsistent data. Before this PR, repo secrets will keep owner_id and repo_id, after this PR, only repo_id >0 but owner_id is 0. Why should we change owner id = 0 here?

Reply:
In database, the secrets works actually like this : 

- user level : ownerId: 15, repoId: 0
- org level : ownerId: 18, repoId: 0
- repo level : ownerId: 0, repoId: 54

Today, the owner is always 0 when repoId is set.

By this PR, I added : 

- instance level: ownerId: 0, repoId: 0

routers/api/v1/repo/action.go file only treats the repo level. Other files are doing the other levels (routers/api/v1/org/secrets.go, routers/api/v1/user/action.go, , routers/api/v1/admin/action.go).

The owner.ID was always set to 0, as it should never be set to something else. I replaced it by an actual "0" to make sure everybody understands it should be 0, always. This part is (and was already) covered by unit tests :)